src/soundio/soundmanagerconfig.cpp

```diff
@@ -409,9 +410,8 @@ unsigned int SoundManagerConfig::getFramesPerBuffer() const {
         audioBufferSizeIndex = kDefaultAudioBufferSizeIndex;
     }
     unsigned int framesPerBuffer = 1;
-    double sampleRate = m_sampleRate; // need this to avoid int division
     // first, get to the framesPerBuffer value corresponding to latency index 1
-    for (; framesPerBuffer / sampleRate * 1000 < 1.0; framesPerBuffer *= 2) {
+    for (; framesPerBuffer / m_sampleRate.toDouble() * 1000 < 1.0; framesPerBuffer *= 2) {
```

Comment:
does this need to be double math? (Not tested)
```suggestion
    for (; framesPerBuffer / (m_sampleRate * 1000) < 1; framesPerBuffer *= 2) {
```
The compiler should be able to move m_sampleRate * 1000 in front of the loop.

Reply:
That looks good.